src/main/java/com/zombiecastlerush/util/Parser.java

```diff
@@ -6,7 +6,7 @@
 
 public class Parser {
 
-    final static List<String> ALLOWED_ACTIONS = Arrays.asList("go", "look", "pick-up", "drop", "attempt","display");
+    final static List<String> ALLOWED_ACTIONS = Arrays.asList("go", "look", "pick-up", "drop", "attempt","display","quit");
```

Comment:
It's minor, but please put a space between after the comma and before your next argument, lol. Like "pick-up", then space, then "drop", and not how it's done later with display and quit.

Alternatively just use cmd-opt-L in intellij.